docs/DeveloperGuide.md

```diff
@@ -367,6 +367,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
    
    Use case ends.
   
+**Use case: Edit tag*
+  
+**MSS**
+  
+1. User requests to list tags
+2. Projact shows a list of tags
+3. User requests to edit a specific tag in the list
+4. Projact edits the tag
+   
+   Use case ends.
+     
+**Extensions**
+  
+* 2a. The list is empty.
+     
+     Use case ends.
+     
+* 3a. The given index is invalid.
+
+    * 3a1. Projact shows an error message.
+
+      Use case resumes at step 2.
+     
```

Comment:
should we include a case for when the given tag name is also invalid? (e.g. t/ is missing)